salt/states/x509.py

```diff
@@ -170,27 +170,24 @@
         - backup: True
 
 """
-
-# Import Python Libs
 from __future__ import absolute_import, print_function, unicode_literals
 
 import copy
 import datetime
+import logging
 import os
 import re
 
-# Import Salt Libs
 import salt.exceptions
 import salt.utils.versions
 
-# Import 3rd-party libs
-from salt.ext import six
-
 try:
     from M2Crypto.RSA import RSAError
 except ImportError:
     pass
 
+log = logging.getLogger(__name__)
```

Comment:
This appears to be unused? I guess you wrote debug logs while working on it.

Reply:
Yeah, I did. It's "inoffensive", we can leave it, unless you object :)